controller/test-files/codebase/show/ok_without_auth.golden

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
shouldn't we have a `.json` extension on these 'golden' files ? That would allow the IDE to render the file with the correct syntax coloring, for example.